### PR TITLE
use System.Text.Json instead of Newtonsoft.Json in NetCore3

### PR DESCRIPTION
**Pull request by ghost1372**
NetCore3 Support json so we dont need Newtonsoft.Json
https://devblogs.microsoft.com/dotnet/try-the-new-system-text-json-apis/

### CODE REVIEW

File: src/Shared/HandyControlDemo_Shared/Service/Data/DataService.cs [hunk 1]
```diff
@@ -86,7 +88,11 @@ internal List<AvatarModel> GetContributorDataList()
             try
             {
                 var json = client.DownloadString(new Uri("https://api.github.com/repos/nabian/handycontrol/contributors"));
+#if !Core
                 var objList = JsonConvert.DeserializeObject<List<dynamic>>(json);
+#else
+                var objList = System.Text.Json.JsonSerializer.Deserialize<List<dynamic>>(json);
```
Comment (NaBian) [inline src/Shared/HandyControlDemo_Shared/Service/Data/DataService.cs:94]: core 3.0 does not support for serializing dynamic objects for the time being.

**Review by NaBian — COMMENTED** [review]
I think we can wait and see.

File: src/Shared/HandyControlDemo_Shared/Service/Data/DataService.cs (hunk 1 above)
Comment (ghost1372) [reply src/Shared/HandyControlDemo_Shared/Service/Data/DataService.cs:94]: @Nabian  Update to net core 3 preview 7 released  yesterday

Comment (NaBian) [reply src/Shared/HandyControlDemo_Shared/Service/Data/DataService.cs:94]: yeah, but with preview 7, an exception will be thrown here.

**Comment by ghost1372** [comment]
https://github.com/dotnet/corefx/issues/38007

**Pull request closed without merging**